algorithms/CPlusPlus/Graphs/prim-algorithm.cpp

```diff
@@ -0,0 +1,144 @@
+#include <limits.h>
+
+#include <iostream>
+#include <set>
+#include <vector>
+#include <map>
+
+#define MAXNUMVERTICES 25
```

Comment:
it is easier to read if you use underscores.
```suggestion
#define MAX_NUM_VERTICES 25
```